src/tools/index.md

```diff
@@ -82,6 +82,9 @@ thanks to the Dart community.
 </li>
 </ul>
 
+A [Language Server Protocol implementation](https://github.com/dart-lang/sdk/blob/master/pkg/analysis_server/tool/lsp_spec/README.md)
+is also available for LSP-capable editors that do not have specific Dart extensions.
```

Comment:
Maybe hyper-link `LSP-capable editors` to https://microsoft.github.io/language-server-protocol/implementors/tools/ ?

Reply:
Good idea, done!